ui/shared/constants/sampleConstants.js

```diff
@@ -0,0 +1,2 @@
+
+export const DATASET_TYPE_VARIANT_CALLS = 'VARIANTS'
```

Comment:
why not just put this in the main constants file?

Reply:
oops, good point